images2gif.py

```diff
@@ -0,0 +1,1095 @@
+# -*- coding: utf-8 -*-
+#   Copyright (C) 2012, Almar Klein, Ant1, Marius van Voorden
```

Comment:
I wrote it in the commit:
Original fork (Python 3): https://github.com/isaacgerg/images2gif

It's a fork compatible with Py 3+, unfortunately I had to make some custom chages to make it work with frames produced by ImageSequencer.

Here is my fork:
https://github.com/CasualBeer/images2gif

Should I mark it in the code ? (I'm new to github and not sure what are the conventions in public repos)